crits/dashboards/views.py

```diff
@@ -1,17 +1,33 @@
+import json
+
 from django.contrib.auth.decorators import user_passes_test
-from crits.core.crits_mongoengine import json_handler
-from crits.core.user_tools import user_can_view_data
+from django.core.urlresolvers import reverse
+from django.http import HttpResponse
 from django.shortcuts import render_to_response
 from django.template import RequestContext
-from crits.core.handlers import generate_global_search, generate_dashboard
-from crits.core.views import dashboard
-from django.http import HttpResponse
-from crits.dashboards.dashboard import SavedSearch, Dashboard
-from crits.dashboards.handlers import toggleTableVisibility,get_saved_searches_list,get_dashboard, getHREFLink, get_obj_type_from_string, clear_dashboard, save_data, get_table_data, generate_search_for_saved_table, delete_table, getRecordsForDefaultDashboardTable
-from crits.dashboards.handlers import switch_existing_search_to_dashboard, add_existing_search_to_dashboard, renameDashboard,changeTheme, deleteDashboard,getDashboardsForUser,createNewDashboard, setDefaultDashboard, cloneDashboard, setPublic, updateChildren
-import json
-import re
-from django.core.urlresolvers import reverse
+
+from crits.core.user_tools import user_can_view_data
+from crits.core.handlers import generate_global_search
+from crits.dashboards.dashboard import Dashboard
+from crits.dashboards.handlers import (toggleTableVisibility,
+                                       get_saved_searches_list,get_dashboard,
+                                       clear_dashboard,
+                                       save_data,
+                                       get_table_data,
+                                       generate_search_for_saved_table,
+                                       delete_table,
+                                       getRecordsForDefaultDashboardTable)
+from crits.dashboards.handlers import (switch_existing_search_to_dashboard,
```

Comment:
Ew giant whitespace blocks. Don't need to repeat the "from crits.dashboards.handlers import" which was just there again to break up the wide line of imported functions